integration-tests/integration/coordinator/remove-involved-staff.spec.js

```diff
@@ -85,17 +86,18 @@ context('A use of force coordinator can remove involved staff', () => {
     viewStatementsPage = ViewStatementsPage.verifyOnPage()
     viewStatementsPage.return().click()
 
-    allIncidentsPage = AllIncidentsPage.verifyOnPage()
-    allIncidentsPage.getNoTodoRows().should('exist')
-    allIncidentsPage.getCompleteRows().should('have.length', 1)
+    notCompletedIncidentsPage = NotCompletedIncidentsPage.verifyOnPage()
+    notCompletedIncidentsPage.getNoTodoRows().should('not.exist')
```

Comment:
this doesn't seem right - are you actually on the NotCompletedPage here? 
I think the "no incomplete rows" should show after making the report complete.

Reply:
Yes, you are correct. I  have moved this check to the end of the test.